### PR TITLE
Create Blog “benefits-of-the-platform-level-data-model-for-firmware-update-standard”

### PR DESCRIPTION
**Pull request by donzef**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  - ilo-resful-api
+  - Redfish
+---
+## Introduction
+
+In 2016, the [Distributed Management Task Force](https://dmtf.org) published the first version of the Platform Level Data Model for Firmware Update Specification ([PLDM for FWUPD](https://www.dmtf.org/dsp/DSP0267)). This DMTF standard started to be implemented in the Redfish® service of HPE iLO 5 version 2.30.
+
+The best definition of PLDM for FWUPD is provided in the introduction paragraph of the [DSP0267](https://www.dmtf.org/sites/default/files/standards/documents/DSP0267_1.1.0.pdf) specification document:
+
+"*The Platform Level Data Model (PLDM) Firmware Update Specification defines messages and data structures for updating firmware or other code objects maintained within the firmware devices of a platform management subsystem. Additional functions related to the sequence of identifying and transferring the firmware, are also defined.*"
```
Comment (drensing) [inline content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md:16]: I just noticed that you used quotation marks here and I don't believe you did in the previous blog post. It's a good idea to do so, though sometimes quotes are indicated simply through italitization. Can you take input from me here to put it into your other blog post?

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+In this type of firmware update, the firmware can be flashed to its destination component by the iLO and become active without any server reboot required.
+
+Examples of firmware falling in this category:
+
+* iLO firmware
+* Power Supply firmware
+* PLDM firmware
+
+> Note: The last bullet of this list is the reason of this article.
```
Comment (drensing) [inline content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md:44]: Now you have me really intrigued... but you leave me hanging. Please... tell me more here!

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+### Type D
+
+UEFI and run time agents, like the Smart Update Manager (SUM) or the Smart Update Tool (SUT)) can flash the firmware content. This type of firmware update may require a system restart.
+
+Examples:
+
+* Smart Array controller firmware
+* Network Interface Cards (NICs) firmware
+
+> Note: devices depending associated to a type D firmware update are called *type-D devices*.
```
Comment (drensing) [inline content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md:73]: I don't understand what this note is saying. "Devices depending..." on what?

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+## Added value of PLDM for FWUPD
+
+When PLDM for FWUPD is not implemented in a Redfish service (i.e. iLO 5 firmware prior to version 2.30), or, if a type D firmware package does not support PDLM for FWUPD, the target device is flashed by a run time agent (SUM or SUT). If no run time agent is present, the firmware cannot be updated as explained in the second  [part](https://developer.hpe.com/blog/hpe-firmware-updates-part-2-interaction-in-operating-modes/) of the above blog post list.
+
+When PLDM for FWUPD is supported by both the iLO Redfish service and a firmware package of a type-D device, the firmware package falls in type A. As a result the iLO can flash the bits and no server reboot is required to activate them anymore.
```
Comment (drensing) [inline content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md:79]: Add a comma after "As a result,"

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+![Partial extract of a PLDM for FWUPD payload.json file](/img/payloadofpldmimage.png "Partial extract of a PLDM for FWUPD payload.json file")
+
+## Conclusion
+
+By  transforming type-D devices into type-A, the Platform Level Data Model for Firmware Update specification provides a real benefit to end customers as no more server reboot is required. Firmware providers, too, take advantage of this new standard, as they don't need to adapt their flash tools to each computer integrators supporting their devices. Finally, computer makers don't need to provide run time agents supporting firmware provider flash tools.
```
Comment (drensing) [inline content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md:96]: I think you're missing a word in the second sentence:
"Firmware providers, too, **can** take advantage..."

The last part of that sentence is a little confusing. Consider the following rewrite:
"Firmware providers, too, can take avantage of this new standard, as they don't need to adapt their flash tools to each computer integrators' supported devices."

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+In addition to the drastic simplification and enhancement of firmware management, the PLDM DMTF standard enhances and simplifies external device supplied devices, in terms of monitoring and configuration. Read the [PLDM for RDE](TBD) article to better understand how.
+
+As previously mentioned, 
+Don't forget to check out some of my other [blog posts](https://developer.hpe.com/search/?term=donze) on the HPE Developer portal to learn more about Redfish tips and tricks.
```
Comment (drensing) [inline content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md:101]: In the preview, these two lines show as one sentence, so you have:
As previously mentioned, Don't forget

I think you meant to do that, but you should change "Don't" to "don't"

**Review by drensing — COMMENTED** [review]
Very nice article. Not many edits.

Thank you, Francois!

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md (hunk 1 above)
Comment (donzef) [reply content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md:16]: Done.

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md (hunk 2 above)
Comment (donzef) [reply content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md:44]: Removed the "> Note:", as it generates too much confusion.

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md (hunk 3 above)
Comment (donzef) [reply content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md:73]: Removed word "depending"

File: content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md (hunk 6 above)
Comment (donzef) [reply content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md]: Done.

**Comment by donzef** [comment]
Ready to publish

**Pull request merged**